projects/step-frontend/src/lib/os-plugins/modules/script-editor/components/script-editor/script-editor.component.scss

```diff
@@ -1,15 +1,27 @@
 :host {
   display: block;
   padding: 1.6rem;
-  width: 100%;
+  max-width: 100%;
 }
 
 mat-card-header {
   display: flex;
   justify-content: space-between;
   align-items: center;
+
+  .color-green {
+    color: green;
+  }
+
+  .color-black {
+    color: black;
+  }
 }
 
 .editor {
   min-height: 60rem;
 }
+
+.mr2 {
```

Comment:
same comment as with the colors, this naming is more of a tailwind or bootstrap style

Reply:
changed class name to "left-btn-group"